docs/logagent/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+title: Logagent Release Notes
+description: Features and improvements releases for Sematext Logagent
+
+Logagent is a dynamic project, and releases are typically in a high frequency. 
```

Comment:
I think this is some old version of LA release notes because I know I adjusted this and I believe Stefan has already merged it.  I don't know any more details.  CC @adnanrahic & @megastef.